dotcom-rendering/src/components/ShareButton.importable.tsx

```diff
@@ -118,13 +118,13 @@ export const CopyNativeShareButton = ({
 			priority="tertiary"
 			iconSide="left"
 			icon={isCopied ? <SvgCheckmark /> : <SvgShareWeb />}
-			css={[
+			cssOverrides={css([
```

Comment:
`cssOverrides` can accept an array of `SerializedStyles`, but they can't be undefined which means conditionals like `isLiveBlogMeta && liveBlogMobileMeta(isCopied)` cause a type error. To work around this we pass the array through Emotion's `css` function to get a single `SerializedStyles` object back.